dm/dm.go

```diff
@@ -142,6 +161,10 @@ func execute() {
 
 	switch args[0] {
 	case "templates":
+		if len(args) < 2 {
+			fmt.Fprintln(os.Stderr, "No registry name supplied")
```

Comment:
We used to use the default registry here. Why do we now require a registry to specified explicitly?